### PR TITLE
get and display list of workers from json server

### PR DESCRIPTION
**Pull request by hrechkovski**

### CODE REVIEW

File: src/pages/listWorkers/ListWorkers.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { Worker } from "../worker/Worker";
+
+export const ListWorkers = () => {
+  const [workers, setWorkers] = useState([]);
```
Comment (mrt123) [inline src/pages/listWorkers/ListWorkers.jsx]: if its not too much problem `[workerdsData, setWorkersData]`

File: src/pages/listWorkers/ListWorkers.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        setWorkers(data);
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  }, []);
+
+  return (
+    <article>
+      {workers.map((worker) => (
```
Comment (mrt123) [inline src/pages/listWorkers/ListWorkers.jsx]: please move mapping outside of JSX

File: src/pages/listWorkers/ListWorkers.jsx (hunk 1 above)
Comment (hrechkovski) [reply src/pages/listWorkers/ListWorkers.jsx]: done

File: src/pages/listWorkers/ListWorkers.jsx (hunk 2 above)
Comment (hrechkovski) [reply src/pages/listWorkers/ListWorkers.jsx]: done

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { ListWorkers } from "./pages/listWorkers/ListWorkers";
+
+function App() {
+  return (
+    <div>
+      <h1>List of current workers in Radpol</h1>
+      <ListWorkers />
```
Comment (mrt123) [inline src/App.jsx]: `<ListWorkers />` should be WorkersList to be gramatically correct

File: src/pages/listWorkers/ListWorkers.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { Worker } from "../worker/Worker";
+
+export const ListWorkers = () => {
+  const [workersData, setWorkersData] = useState([]);
+  const workersDataRender = workersData.map((worker) => (
```
Comment (mrt123) [inline src/pages/listWorkers/ListWorkers.jsx]: workerElemets is a more accurate name then `workersDataRender`

**Review by mrt123 — APPROVED** [review]
There is nothing functionally wrong with this PR.
Please update naming before merging. Please squash commits optionally.

**Pull request merged**